CONTRIBUTING.md

```diff
@@ -0,0 +1,282 @@
+# How to contribute
+
+## How to prepare
+
+* You need a [GitHub account](https://github.com/signup/free)
+* Submit an [issue ticket](https://github.com/tomato42/tlslite-ng/issues) for
+  your issue if there is none yet
+  * Describe the issue and include steps to reproduce if it's a bug, mention
+    the earliest version that you know is affected and the version you're using.
+  * Describe the enhancement and your general ideas on how to implement it
+    if you want to add new feature or extend existing one. This is not
+    necessary if the change is small.
+* If you are able and want to fix the issue, fork the repository on GitHub and
+  see [Developing changes](#developing-changes) section
+
+## Ways to contribute
+
+* Look for issues in the
+  [projects](https://github.com/tomato42/tlsfuzzer/projects) page.
+  [TLS 1.3 coverage](https://github.com/tomato42/tlsfuzzer/projects/1) is the
```

Comment:
Doesn't really fit into contribution guidelines. It would seem very awkward to have that two years from now.